src/o365/spo/commands/contenttypehub/contenttypehub-get.ts

```diff
@@ -0,0 +1,117 @@
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import auth from '../../../../Auth';
+import {
+  CommandOption,
+  CommandValidate,
+  CommandError
+} from '../../../../Command';
+import SpoCommand from '../../../base/SpoCommand';  
+import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+}
+
+class SpoContentTypeHubGetCommand extends SpoCommand {
+
+  public get name(): string {
+    return commands.CONTENTTYPEHUB_GET;
+  }
+
+  public get description(): string {
+    return 'Return the URL of the SharePoint Content Type Hub of the Tenant';
+  }
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+    if (!auth.service.connected) {
```

Comment:
There was an issue in the CLI where for some reason SPO commands would allow to execute without checking if you were logged in and would then prompt you for auth themselves. It's fixed in one of the PRs, but what you said stands: there is no need to check for connection state in the command itself as it's done before already.